spritz/app/src/main/res/layout/main.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  android:background="@android:color/black"
+  android:orientation="vertical">
+
+  <com.airbnb.lottie.LottieAnimationView
+    android:id="@+id/animation_view"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_centerInParent="true"
+    app:lottie_autoPlay="false"
+    app:lottie_fileName="demo-pager.json" />
+
+  <android.support.v4.view.ViewPager
+    android:id="@+id/viewpager"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent" />
+
+  <LinearLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="horizontal">
+
+    <Button
+      android:id="@+id/btn_previous"
+      android:layout_width="0dp"
+      android:layout_height="60dp"
+      android:layout_weight="1"
+      android:text="@string/previous" />
+
+    <Button
+      android:id="@+id/btn_continue"
+      android:layout_width="0dp"
+      android:layout_height="60dp"
+      android:layout_weight="1"
+      android:text="@string/next" />
+
+  </LinearLayout>
+
```

Comment:
Extra whiteline

Reply:
Removed whiteline in b8aac679c88dfbfb33f64c4fbb71788c19578ac9.